### PR TITLE
MC labels implemented; bug in optical properties fixed 

### PR DESCRIPTION
**Pull request by AllaMaevskaya**

### CODE REVIEW

File: Common/Field/include/Field/MagneticWrapperChebyshev.h [hunk 1]
```diff
@@ -180,24 +180,24 @@ class MagneticWrapperChebyshev : public TNamed
       return mMaxRadiusTPCRat;
     }
```
Comment (sawenzel) [inline Common/Field/include/Field/MagneticWrapperChebyshev.h]: why is this file changed / committed?

File: Common/MathUtils/include/MathUtils/Chebyshev3D.h [hunk 2]
```diff
@@ -27,7 +27,8 @@ class TMethodCall;  // lines 16-16
```
Comment (sawenzel) [inline Common/MathUtils/include/MathUtils/Chebyshev3D.h]: why is this file changed / committed? Please only commit things related to your own developments.

Comment (AllaMaevskaya) [reply Common/MathUtils/include/MathUtils/Chebyshev3D.h]: Sorry, I run clang and then did commit .
How to reverse this?

Comment (shahor02) [reply Common/MathUtils/include/MathUtils/Chebyshev3D.h]: Since it is in your last commit, do:
cd O2
git reset HEAD~1
git checkout Common/Field/include/Field/MagneticWrapperChebyshev.h
git checkout Common/MathUtils/include/MathUtils/Chebyshev3D.h 
git checkout Common/MathUtils/include/MathUtils/Chebyshev3DCalc.h
git checkout Common/MathUtils/include/MathUtils/MathBase.h
git checkout  Detectors/TPC/calibration/src/CalibPulser.cxx

git commit -a -m "MC labels implemented; bug in optical properties fixed"

git push -f AllaMaevskaya fit-common-v0

How are you applying the clang-format?

Comment (AllaMaevskaya) [reply Common/MathUtils/include/MathUtils/Chebyshev3D.h]: git clang-format HEAD~10

Comment (shahor02) [reply Common/MathUtils/include/MathUtils/Chebyshev3D.h]: Why 10? You had only 4 commits
